### PR TITLE
Hide an empty list when there are no items in `Holdings` record

### PR DESCRIPTION
**Pull request by SergiyVSergiyenko**
https://issues.folio.org/browse/UIIN-1076

## Purpose
Do not show an empty list if there are no items in the given `Holdings` record, and the draggable mode is deactivated.

### Demo

Hide_empty_list

### CODE REVIEW

File: src/Instance/HoldingsList/Holding/Holding.js [hunk 1]
```diff
@@ -98,7 +101,8 @@ const Holding = ({
       }
       <Accordion
         id={holding.id}
-        closedByDefault={false}
+        open={open}
+        onToggle={() => setOpen(!open)}
```
Comment (NikitaSedyx) [inline src/Instance/HoldingsList/Holding/Holding.js:105]: please don't create functions in props

File: src/Instance/ItemsList/ItemsListContainer.js [hunk 2]
```diff
@@ -3,20 +3,24 @@ import React, {
...
 } from 'react';
 import PropTypes from 'prop-types';
+import { isEmpty } from 'lodash';
 
 import { stripesConnect } from '@folio/stripes/core';
 
 import ItemsList from './ItemsList';
 
-const ItemsListContainer = ({ holding, mutator, ...rest }) => {
+const ItemsListContainer = ({ holding, mutator, setOpen, ...rest }) => {
```
Comment (NikitaSedyx) [inline src/Instance/ItemsList/ItemsListContainer.js:12]: it's not obvious from his component what `setOpen`  is for

File: src/Instance/HoldingsList/Holding/Holding.js (hunk 1 above)
Comment (SergiyVSergiyenko) [reply src/Instance/HoldingsList/Holding/Holding.js]: Thanks for pointing that out. Fixed.

File: src/Instance/ItemsList/ItemsListContainer.js (hunk 2 above)
Comment (SergiyVSergiyenko) [reply src/Instance/ItemsList/ItemsListContainer.js:12]: `setOpen` function controls the expanding/collapsing of the accordion depending on the presence/absence of elements in it.

**Pull request merged**